agora/Agora/Utils.hs

```diff
@@ -407,3 +421,42 @@ pisDNothing = phoistAcyclic $
     flip pmatch $ \case
       PDNothing _ -> pconstant True
       _ -> pconstant False
+
+-- | @since 1.0.0
```

Comment:
Please add short comment about the difference between `psymbolValueOf` and this function.